skare3_tools/config.py

```diff
@@ -103,11 +107,21 @@
 
 
 def _app_data_dir_():
-    ska_data_dir = os.path.join(os.environ["SKA"], "data", "skare3", "skare3_data")
+    home_dir = os.path.expanduser("~")
```

Comment:
I think this might be pathlib.Path.home() , but figure you will probably have merge conflicts on #109 anyway and can look at the places you want to remove os.path again.